Source/bmalloc/bmalloc/VMAllocate.h

```diff
@@ -139,60 +180,39 @@ inline void* vmAllocate(size_t vmSize, VMTag usage = VMTag::Malloc)
 inline void vmDeallocate(void* p, size_t vmSize)
 {
     vmValidate(p, vmSize);
+#if BPLATFORM(WIN)
+    auto success = VirtualFree(p, vmSize, MEM_DECOMMIT);
```

Comment:
This is not the same to `munmap`. It keeps virtual address region. This is rather, decommit.
You can check `OSAllocatorWin.cpp` in WTF, it implements most of these operations.
https://devblogs.microsoft.com/oldnewthing/20170113-00/?p=95185